### DISCUSSION TITLE
Microfrontend

### DISCUSSION THREAD

**Discussion by softshipper** [vercel/next.js#23408] [Help]
Hi all   

I would like to use micro frontend architecture to build web apps based on NextJS.  
I consider to use https://www.npmjs.com/package/@module-federation/nextjs-mf but not sure, if it is production ready or not. 
Any suggestions?  

Thanks

**Comment by vvenv** [comment]
May be just use [Multi Zones](https://nextjs.org/docs/advanced-features/multi-zones)
